app/views/admin/index.html.erb

```diff
@@ -17,6 +17,6 @@
 <h2>Incomplete Invoices</h2>
 <ul>
   <% @incomplete_invoices.each do |invoice| %>
-    <li><%= link_to( "Invoice #{invoice.id}", "/admin/invoices/#{invoice.id}" ) %></li>
+    <li><%= link_to( "Invoice #{invoice.id}", "/admin/invoices/#{invoice.id}" ) %> <%= "Created on: #{invoice.created_at.strftime('%A, %B %d, %Y')}"%></li>
```

Comment:
Looks like ruby does the truck for converting the date - one other option may be using `.to_formatted_s`:
https://apidock.com/rails/Time/to_formatted_s

I haven't used it, so I'm not sure if it'll work (or maybe you already considered it and it didn't do the trick?), but perhaps worth checking out.